web/src/beta/features/Dashboard/type.ts

```diff
@@ -38,6 +38,7 @@ export type User = {
   name: string;
   email?: string;
 };
+export type Role = "OWNER" | "READER" | "WRITER" | "MAINTAINER";
```

Comment:
have we considered using the `Role` type from the generated GraphQL types in `@reearth/services/gql`